src/inspector/internal/services/elementrepositoryservice.cpp

```diff
@@ -394,10 +398,10 @@ QList<mu::engraving::EngravingItem*> ElementRepositoryService::findTremolos() co
     QList<mu::engraving::EngravingItem*> resultList;
 
     for (mu::engraving::EngravingItem* element : m_exposedElementList) {
-        if (element->isTremolo()) {
+        if (element->isType(ElementType::TREMOLO_TWOCHORD)) {
```

Comment:
It seems confusing to me that the `findTremolos` method, which is used in the case of `ElementType::TREMOLO_SINGLECHORD` too, only finds two chord tremolos...

In particular, this causes `TremoloSettingsModel::requestElements()` to find two-note tremolos twice and one-note tremolos not at all.

I would propose to rename it to `findTwoChordTremolos`, and do something different for `ElementType::TREMOLO_SINGLECHORD`.

Reply:
This method can find both types of tremol
But I made a limitation to only two chords, because of the comment and the condition that at the moment we can only change tremolos with `customStyleApplicable()`. Single-chord tremolos do not have this property, so they have not been returned using this method before.

Reply:
But I think the problem is that a method that can find both types of tremolos is not useful (anymore). 

The result is namely that `ElementRepositoryService::findElementsByType(ElementType::TREMOLO_SINGLECHORD)` returns a list that may contain two chord tremolos, which is unexpected.

(And theoretically, `ElementRepositoryService::findElementsByType(ElementType::TREMOLO_TWOCHORD)` returns a list that may contain single chord tremolos, but that's not currently a problem in practice, because of the `customStyleApplicable()` thing that you mention.)

So, in `TremoloSettingsModel::requestElements()`, `m_repository->findElementsByType(ElementType::TREMOLO_SINGLECHORD)` and `m_repository->findElementsByType(mu::engraving::ElementType::TREMOLO_TWOCHORD)` return exactly the same list; as a result, `m_elementList` will contain every element twice.

Reply:
I understood the problem, thanks!
I rewrote the logic, moved the condition from the repository to the model, I think it is more appropriate there.

Reply:
Now it looks like `findTwoTremolos` is not used anywhere :)
But the idea looks good to me!

Reply:
removed )